platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/internal/Builder.java

```diff
@@ -486,6 +487,15 @@ public void cancelled(Cancellable cancellable) {
         };
     }
 
+	/**
+	 * Modify the environment variables that are set for a command being executed.
+	 * 
+	 * @param environmentVars The environment variables of the command process.
+	 * @param configuration Connfiguration of the build task.
+	 */
+	protected void modifyCommandEnvironment(Map<String, String> environmentVars, BuilderConfiguration configuration) {
```

Comment:
good change! 
can we have a logger passed to this method as well , so we can comment any change in the build log